write-fonts/src/tables/post.rs

```diff
@@ -14,21 +14,30 @@ pub struct PString(String);
 impl Post {
     /// Construct a new version 2.0 table from a glyph order.
     pub fn new_v2<'a>(order: impl IntoIterator<Item = &'a str>) -> Self {
-        let known_glyphs = read_fonts::tables::post::DEFAULT_GLYPH_NAMES
+        let standard_glyphs = read_fonts::tables::post::DEFAULT_GLYPH_NAMES
             .iter()
             .enumerate()
             .map(|(i, name)| (*name, i as u16))
             .collect::<HashMap<_, _>>();
+        const NUM_STANDARD: usize = 258;
```

Comment:
I would write this as, 
```suggestion
        const NUM_STANDARD: usize = read_fonts::tables::post::DEFAULT_GLYPH_NAMES.len();
```

because it communicates more clearly what is going on.

Reply:
Thanks, I'll make this change later in another PR.